charts/argo-cd/Chart.yaml

```diff
@@ -16,9 +16,9 @@ maintainers:
   - name: seanson
 dependencies:
   - name: redis-ha
-    version: 4.12.17
+    version: 4.13.0
     repository: https://dandydeveloper.github.io/charts/
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Changed]: Make AWS ALB GRPC backend protocol version configurable"
+    - "[Changed]: Update to the latest redis-ha chart which defaults to redis 6.2.5"
```

Comment:
I believe the changelog should append, not replace. 🙂

Reply:
It's only keeping changes relevant to this version, I believe this is okay

Reply:
Ah, I thought it meant a changelog per "major" verison, not "minor".